openstack-typescript/.gitignore

```diff
@@ -0,0 +1,3 @@
+/bin/
+/node_modules/
+/.pulumi/
```

Comment:
Mind deleting this line?  I'm removing the other copies in #32, it is no longer needed.

Reply:
Sure no problem.